plugin-cgroupv1/src/oar2/utils.rs

```diff
@@ -0,0 +1,16 @@
+use alumet::resources::ResourceConsumer;
+use std::fs::File;
+
+#[derive(Debug)]
+pub struct Cgroupv1Resources {
```

Comment:
I'm still not sure about the name, there's no resource here... How about `OpenedCgroupv1`? :thinking: It's a struct about a cgroup v1 that we have "opened" and prepared for metric collection.